src/spikeinterface/core/base.py

```diff
@@ -846,43 +846,63 @@ def save_to_memory(self, sharedmem=True, **save_kwargs) -> "BaseExtractor":
         return cached
 
     # TODO rename to saveto_binary_folder
-    def save_to_folder(self, name=None, folder=None, overwrite=False, verbose=True, **save_kwargs):
+    def save_to_folder(
+        self,
+        name: str = None,
+        folder: str | Path = None,
+        overwrite: str = False,
+        verbose: bool = True,
+        **save_kwargs,
+    ):
         """
-        Save extractor to folder.
+        Save the extractor and its data to a folder.
 
-        The save consist of:
-          * extracting traces by calling get_trace() method in chunks
-          * saving data into file (memmap with BinaryRecordingExtractor)
-          * dumping to json/pickle the original extractor for provenance
-          * dumping to json/pickle the cached extractor (memmap with BinaryRecordingExtractor)
+        This method extracts trace data, saves it to a file (using a memory-mapped format
+        with BinaryRecordingExtractor), and stores both the original extractor's provenance
```

Comment:
```suggestion
        This method extracts trace data, saves it to a file (using the memory-mapped
        BinaryRecordingExtractor format), and stores both the original extractor's provenance
```
would probably be the slightly more standard short way to write this.

Reply:
Thanks, once we decide on the Path name I will batch all of these suggestions.